Doc/make.bat

```diff
@@ -152,6 +152,7 @@ goto end
 if NOT "%2" EQU "" (
     echo.Can't specify filenames to build with htmlview target, ignoring.
 )
+set SPHINXOPTS=-D html_theme_options.body_max_width=none %SPHINXOPTS%
```

Comment:
htmlview is the wrong target; it makes html in the default location and  views the result in the default browser.
But the new line works in that the line limit is gone in the resulting html.  So the new line just needs to go somewhere else.

I tried moving the new line up under 136 `if "%1" EQU "htmlhelp" (` but that failed.  I believe this is because the html has already been built by Sphinx with the line limit and the next step just compresses the html as is into the .chm.  I will try putting the addition guarded by the same `if` somewhere up above.

Reply:
I found the right place, a couple of line up, before `cmd...`.  Preparing blurb.

Reply:
Sorry about that, I managed to misread pretty terribly.  Fixed; still not tested.